### PR TITLE
add Marker to flathub

### PR DESCRIPTION
**Pull request by jsparber**
For reference https://github.com/fabiocolacio/Marker/issues/40

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.github.fabiocolacio.marker

File: com.github.fabiocolacio.marker.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "app-id": "com.github.fabiocolacio.marker",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "/bin/com.github.fabiocolacio.marker",
```
Comment (Barthalion) [inline com.github.fabiocolacio.marker.json]: It should be just a filename, not full path.

File: com.github.fabiocolacio.marker.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc", "--socket=x11",
+    /* Wayland access */
+    "--socket=wayland",
+    /* We want full fs access so we can read the files */
+    "--filesystem=host"
```
Comment (Barthalion) [inline com.github.fabiocolacio.marker.json]: Sounds like `home` would be enough?

**Comment by Barthalion** [comment]
So we don't forget: should someone besides you have commit rights to the repository?

**Comment by nedrichards** [comment]
Yeah, sounds like the upstream maintainer should as well - from the linked thread - even if they don't want to spend a lot of time on it

**Comment by jsparber** [comment]
Oh, didn't thought that i would get ownership, actually only https://github.com/fabiocolacio should have commit rights, he is the maintainer.

**Comment by jsparber** [comment]
made the changes you proposed

**Comment by TingPing** [comment]
> Oh, didn't thought that i would get ownership, actually only https://github.com/fabiocolacio should have commit rights, he is the maintainer.

Considering you are doing the bulk of the Flatpak work it would be logical for you to help co-maintain it.

File: com.github.fabiocolacio.marker.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "com.github.fabiocolacio.marker",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc", "--socket=x11",
+    /* Wayland access */
+    "--socket=wayland",
+    /* We want full fs access so we can read the files */
+    "--filesystem=home"
```
Comment (TingPing) [inline com.github.fabiocolacio.marker.json]: In theory this isn't needed, it only opens a single file at a time right?

Comment (jsparber) [reply com.github.fabiocolacio.marker.json]: When i remove it i can't see any files in the open dialog

Comment (TingPing) [reply com.github.fabiocolacio.marker.json]: Ah, Upstream should start using this API https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html for its dialogs.

Comment (jsparber) [reply com.github.fabiocolacio.marker.json]: Ok, i made a issue upstream https://github.com/fabiocolacio/Marker/issues/133

**Comment by nedrichards** [comment]
bot, build com.github.fabiocolacio.marker

File: com.github.fabiocolacio.marker.json (hunk 3 above)
Comment (jsparber) [reply com.github.fabiocolacio.marker.json]: It seams they did already implement this. (I didn't test it yet)
https://github.com/fabiocolacio/Marker/commit/489cec84a0b7932af4203dc3aacff50a8399530e

File: com.github.fabiocolacio.marker.json (hunk 1 above)
Comment (jsparber) [reply com.github.fabiocolacio.marker.json]: Fixxed in https://github.com/flathub/flathub/pull/249/commits/49f949ce2e39c1035d4b4a9a308827452a8dc6ae

File: com.github.fabiocolacio.marker.json (hunk 2 above)
Comment (jsparber) [reply com.github.fabiocolacio.marker.json]: https://github.com/flathub/flathub/pull/249/commits/49f949ce2e39c1035d4b4a9a308827452a8dc6ae

**Comment by nedrichards** [comment]
bot, build com.github.fabiocolacio.marker

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.fabiocolacio.marker

**Pull request closed without merging**